### ISSUE TITLE
edit URI changed from MkDocs

### ISSUE THREAD

**Issue by garfieldnate** [timothycrosley/portray#88]
MkDocs uses `edit/master` as the default `edit_uri` value, so without documentation stating otherwise I expected this to be the same in Portray. Changing this value will break clients, but it would be helpful to let the user know that this is happening. A single-line message like, "edit_uri not set by user, defaulting to edit/main" (or edit/default) would be helpful.

**Comment by robbintt** [comment]
Agreed, looks like it is hardcoded here, but would be better if it were configurable, then this project can still set a default of main while allowing others to fall back to `master` when they are unable to change the branch for legacy reasons, etc.

E.g. in PyProject.toml:
```
[tool.portray.mkdocs]
edit_uri = "edit/master"
```
